File: src/main/java/com/thelads/UnicornApplication.java
package com.thelads;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;

@SpringBootApplication
@EnableAsync
public class UnicornApplication {

    public static void main(String[] args) {
        SpringApplication.run(UnicornApplication.class, args);
    }
}

File: src/main/kotlin/com/thelads/UnicornApplication.kt
package com.thelads

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableAsync

@SpringBootApplication
@EnableAsync
open class UnicornApplication

fun main(args: Array<String>) {
    runApplication<UnicornApplication>(*args)
}

